server/pbench/lib/pbench/__init__.py

```diff
@@ -227,8 +228,9 @@ def actions_tracking_closure(cl_actions):
             status = resp['status']
         except KeyError as e:
             assert not ok
-            # resp is not of expected form
-            print(resp)
+            # resp is not of expected form.
+            # Limit the error message to 16K
+            print(resp[:_MAX_ERRMSG_LENGTH])
```

Comment:
Do we have to first convert `resp` into a string to do this?  Perhaps we should use `json.dumps` instead?

And not sure we should put the size we are limiting the error message to in the comments.

Reply:
Done in latest.